tests/ui/test_pvc_ui.py

```diff
@@ -23,22 +23,37 @@ def teardown(self):
     @ui
```

Comment:
since we don't have jobs for the ui tier, these tests should be part of tierx

Reply:
changed, please check